### PR TITLE
Update gems

### PR DESCRIPTION
**Pull request by kairadiagne**
Although we don't support cocoa pods the gemfile.lock still contained cocoa pods and related dependencies. I updated it using bundle install.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,223 +1,13 @@
...
-    xcpretty-json-formatter (0.1.1)
-      xcpretty (~> 0.2, >= 0.0.7)
-    xcpretty-travis-formatter (1.0.0)
-      xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
-  cocoapods
```
Comment (kairadiagne) [inline Gemfile.lock:217]: @AvdLee Do you know why we used to have the gems fastlane, pretty etc in here but now only the dotenv gem?

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/bcd31a09-1ee7-4548-ae5a-1140b090c8a2" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>GitBuddyTests: Executed 57 tests (0 failed, 0 retried, 0 skipped) in 1.115 seconds</td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |
GitBuddy | 0.00% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 204941d9addd0bc76d0e72ed8881ef416351632d
</p>

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 4.1.2](https://github.com/WeTransfer/GitBuddy/releases/tag/4.1.2) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

File: Gemfile.lock (hunk 1 above)
Comment (AvdLee) [reply Gemfile.lock:217]: I would not be surprised if that's a result of copying files from other projects or using a general setup for the project. They can be safely removed at least!

**Pull request merged**